.buildkite/scripts/packaging/prepare-release-manager.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
```

Comment:
As far as I see this file is a copy of `.ci/scripts/prepare-release-manager.sh` but `.ci/scripts/prepare-release-manager.sh` has not been deleted...